src/main/java/com/epam/aidial/cfg/dao/mapper/InterceptorEntityMapper.java

```
@@ -173,11 +194,19 @@ private Pair<List<ApplicationEntity>, List<ModelEntity>> findApplicationsAndMode
         return Pair.of(applications, models);
     }
 
+    private List<ApplicationTypeSchemaEntity> findApplicationTypeSchemaById(List<String> ids) {
+        if (CollectionUtils.isEmpty(ids)) {
+            return List.of();
+        }
+        return Lists.newArrayList(applicationTypeSchemaJpaRepository.findAllById(ids));
```

I think we should throw not found exception if some of ids were not found